storm-client/src/jvm/org/apache/storm/stats/StatsUtil.java

```diff
@@ -1547,26 +1533,21 @@ public static ComponentPageInfo aggCompExecsStats(
      * @param timeout       timeout
      * @return a HashMap of updated executor heart beats
      */
-    public static Map<List<Integer>, Map<String, Object>> updateHeartbeatCache(Map<List<Integer>, Map<String, Object>> cache,
+    public static void updateHeartbeatCache(Map<List<Integer>, Map<String, Object>> cache,
```

Comment:
We're removing the return, so we need to remove it from javadoc too.

Reply:
yeah